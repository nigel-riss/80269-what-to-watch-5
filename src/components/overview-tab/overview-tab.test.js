import React from 'react';
import renderer from 'react-test-renderer';
import OverviewTab from './overview-tab.jsx';
import filmsMock from '../../mocks/films.js';


it(`Should OverviewTab render correctly`, () => {
  const tree = renderer
    .create(
        <OverviewTab
          film={filmsMock[0]}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
