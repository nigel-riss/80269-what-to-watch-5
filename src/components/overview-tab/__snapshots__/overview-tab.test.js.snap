// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OverviewTab render correctly 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      7,5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        274
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      Biopic of billionaire Howard Hughes, starting with his early filmmaking years as owner of R.K.O. Pictures, but mostly focusing on his role in designing and promoting new aircraft. Hughes was a risk-taker spending several fortunes on designing experimental aircraft and eventually founding TWA as a rival to Pan Am airlines owned by his great rival Juan Trippe. When Trippe's politico Senator Ralph Owen Brewster accuses Hughes of being a war profiteer, it's Hughes who gains the upper hand. Hughes also had many women in his life including a long relationship with Katharine Hepburn. From an early age, however, Hughes was also germophobic and would have severe bouts of mental illness.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Martin Scorsese
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Leonardo DiCaprio, Cate Blanchett, Kate Beckinsale, John C. Reilly, Alec Baldwin, Alan Alda, Ian Holm
         and other
      </strong>
    </p>
  </div>,
]
`;
